samples/custom-recoverability/Core_7/Server/MyHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using NServiceBus;
+using NServiceBus.Logging;
+using System;
+using System.Threading.Tasks;
+
+#region MyHandler
+public class MyHandler :
+    IHandleMessages<MyMessage>
+{
+    static ILog log = LogManager.GetLogger<MyHandler>();
+
+    public Task Handle(MyMessage message, IMessageHandlerContext context)
+    {
+        log.Info($"Message received. Id: {message.Id}");
+       // throw new ArgumentNullException("Uh oh - something went wrong....");
+        // throw new DivideByZeroException("DivideByZeroException - something went wrong....");
```

Comment:
Indentation is weird here. Why are these commented out?

Reply:
These are commented out to begin with and the sample doc has instructions when to uncomment them and what happens when its done.